### DISCUSSION TITLE
USAA Login

### DISCUSSION THREAD

**Discussion by wesweberjr** [uBlockOrigin/uAssets#22954] [Q&A]
No problem logging into USAA.com on Edge. However using Floorp to logon to USAA I get this error:

![usaa-error](https://github.com/uBlockOrigin/uAssets/assets/42493322/0838320b-d018-4e21-8239-7e1a45009859)

What setting will allow the USAA website to work.

**Comment by Yuki2718** [comment]
Please report in [Issues](https://github.com/uBlockOrigin/uAssets/issues) with required details
